community/quassel/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Maintainer: Yves Schumann <yves@eisfair.org>
 
 pkgname=quassel
-pkgver=0.12.4
-pkgrel=3
+pkgver=0.12.5
+pkgrel=1
 url="http://www.quassel-irc.org/"
```

Comment:
https://quassel-irc.org/

Reply:
What I meant here was to update the url and source url to the TLS variant (https://)